test/object_store/test_basic.py

```diff
@@ -75,20 +76,24 @@ def kill_with_dir(old_pid, run_dir):
     scylla_link = os.path.join(run_dir, 'test_scylla')
     os.unlink(scylla_link)
 
+class Cluster:
```

Comment:
could you add two empty lines instead a single one between top-level functions? i would suggest to be compliant to PEP8 for better readability. see https://peps.python.org/pep-0008/#blank-lines